apps/builder/.storybook/main.ts

```diff
@@ -0,0 +1,29 @@
+import path from "path";
+import type { StorybookConfig } from "@storybook/react-vite";
+import storybookConfig from "@webstudio-is/storybook-config";
+
+export default {
+  ...storybookConfig,
+  stories: ["../app/**/*.stories.mdx", "../app/**/*.stories.@(jsx|ts|tsx)"],
+  async viteFinal(config) {
+    return {
+      ...config,
+      define: {
+        ...config.define,
+        // storybook use "util" package internally which is bundled with stories
+        // and gives and error that process is undefined
+        "process.env.NODE_DEBUG": "undefined",
+      },
+      resolve: {
+        ...config.resolve,
+        conditions: ["source", "import", "module", "browser", "default"],
```

Comment:
According to the docs you only have to include additional conditions here:
https://github.com/vitejs/vite/blob/f900acdf265b7be9b52b99f04c101e6cf62b51b4/docs/config/shared-options.md?plain=1#L148